chcemvediet/apps/inforequests/models/inforequest.py

```diff
@@ -370,8 +370,9 @@ def disclosure_level(self):
         Cached maximum disclosure level to the inforequest. Returns None if the inforequest hasn't
         got any disclosure action.
         """
-        return Action.objects.of_inforequest(inforequest=self).aggregate(
-                Max(u'disclosure_level'))[u'disclosure_level__max']
+        return (Action.objects.of_inforequest(inforequest=self)
+                .aggregate(Max(u'disclosure_level'))[u'disclosure_level__max']
```

Comment:
@viliambalaz 
Pri takychto retazenych volaniach metod radsej zalamujme pred volanim metody, nie v argumentoch. Nech su argumenty na tom istom riadku ako nazov metody. Tusim aj inde to mame rovnako, ci?